explorer/src/Pos/Explorer/Web/Server.hs

```diff
@@ -711,8 +715,31 @@ topsortTxsOrFail f =
 
 -- | Deserialize Cardano or RSCoin address and convert it to Cardano address.
 -- Throw exception on failure.
-cAddrToAddr :: MonadThrow m => CAddress -> m Address
-cAddrToAddr cAddr@(CAddress rawAddrText) =
+-- cAddrToAddr :: MonadThrow m => CAddress -> m Address
+-- cAddrToAddr cAddr@(CAddress rawAddrText) =
+--     -- Try decoding address as base64. If both decoders succeed,
+--     -- the output of the first one is returned
+--     let mDecodedBase64 =
+--             rightToMaybe (B64.decode rawAddrText) <|>
+--             rightToMaybe (B64.decodeUrl rawAddrText)
+--     in case mDecodedBase64 of
+--         Just addr -> do
+--             -- cAddr is in RSCoin address format, converting to equivalent Cardano address
+--             -- Originally taken from:
+--             -- * cardano-sl/tools/src/keygen/Avvm.hs
+--             -- * cardano-sl/tools/src/addr-convert/Main.hs
+--             unless (BS.length addr == 32) $
+--                 throwM badAddressLength
+--             pure $ makeRedeemAddress $ redeemPkBuild addr
+--         Nothing ->
+--             -- cAddr is in Cardano address format
+--             either badCardanoAddress pure (fromCAddress cAddr)
+--   where
+--     badAddressLength = Internal "Address length is not equal to 32, can't be redeeming pk"
+--     badCardanoAddress = const $ throwM $ Internal "Invalid Cardano address!"
+
+cAddrToAddr :: CAddress -> Address
+cAddrToAddr (CAddress rawAddrText) =
```

Comment:
Why did you change the type of `cAddrToAddr` when it's not even used in this module now? Is it used elsewhere? If so it would need to be fixed, if not it does not need to be exported.

Reply:
It's being used in the test, there are comments in the PR for running the tests.